packages/orchestrator-ui-components/src/pages/processes/WfoProcessDetail.tsx

```diff
@@ -185,13 +190,27 @@ export const WfoProcessDetail = ({
             },
         });
 
+    const docsUrl = workflowInformationLinkUrl + processDetail?.workflowName;
+
     return (
         <>
             <EuiFlexGroup>
                 <EuiFlexItem>
                     <EuiPageHeader pageTitle={pageTitle} />
                     <EuiSpacer />
-                    <EuiText size="s">{productNames}</EuiText>
+                    <EuiFlexGroup gutterSize={'s'} alignItems={'center'}>
+                        {showWorkflowInformationLink && (
+                            <EuiToolTip content={t('openWorkflowTaskInfo')}>
```

Comment:
I'm always a big fan of small components (functions) and less indentation, so personally I would create a WorkFlowToolTip (which might be reused), but that's up to you.